cdk/src/functions/enable_controltower/index.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from awsapilib import ControlTower
+
+
+CREATE = 'Create'
```

Comment:
Are `Delete` and `Update` not relevant anymore?
https://github.com/superwerker/superwerker/blob/main/templates/control-tower.yaml#L56

Reply:
They were not used in the lambda, so I removed them. Also the linter would complain ;)